components/TestSubmissionsResults/TestSubmissionsResults.module.scss

```diff
@@ -0,0 +1,27 @@
+﻿@import '../../pageStyles/constants';
+
+.listItem{
+  font-family: $softwireFont;
+  font-size: $contentFontSize;
+  color: $softwireWhite;
+  @include sharedListStyle;
+}
+
+.textArea {
+  border: solid 2px $softwireYellow;
+  outline: none;
+  width: 290px;
+  height: 120px;
+}
+
+@media (min-width: 768px){
+  .textArea {
+    width: 700px;
+  }
+}
+
+@media (min-width: 1024px) {
+  .textArea {
+    width: 900px;
+  }
+}
```

Comment:
The control of the width seems a little odd.
Won't this make it slightly off-centre at most resolutions and not fill the screen properly?

Why not just make it take up eg `{the full width} - {some padding}`?
Feels like that would require fewer breakpoints and also make a nicer product